libs/sdk-ui-dashboard/src/presentation/widget/insightMenu/DashboardInsightMenu.tsx

```diff
@@ -0,0 +1,19 @@
+// (C) 2020 GoodData Corporation
+import React, { useMemo } from "react";
+
+import { useDashboardComponentsContext } from "../../dashboardContexts";
+import { useDashboardInsightMenuProps } from "./DashboardInsightMenuPropsContext";
+
+/**
+ * @internal
+ */
+export const DashboardInsightMenu = (): JSX.Element => {
+    const { insight, widget } = useDashboardInsightMenuProps();
+    const { InsightMenuComponentProvider } = useDashboardComponentsContext();
+    const InsightMenuComponent = useMemo(
+        () => InsightMenuComponentProvider(insight, widget),
```

Comment:
Question not related to this PR, but is this a good pattern? (imagine edit mode -> insight/widget changes -> then the whole component instance is re-created, instead of simple prop change)